draft-ietf-httpbis-http2bis.xml

```diff
@@ -4206,8 +4211,8 @@ cookie: e=f
             Institute</organization>
             </author>
             <date year="1981" month="September"/>
-	  </front>
-	</reference>
+    </front>
+  </reference>
```

Comment:
Is this whitespace change intentional?

Reply:
No, my editor does a cleanup when I close it, and I caught this.  I made the change on the main branch, so this is redundant (and wrong).